R-package/tests/testthat/test_random.R

```diff
@@ -0,0 +1,19 @@
+require(mxnet)
+
+context("random")
+
+test_that("mx.runif", {
+  X <- mx.runif(shape=50000, min=0, max=4, ctx=mx.ctx.default())
+  expect_equal(X>=0, mx.nd.ones(50000))
+  expect_equal(X<=4, mx.nd.ones(50000))
+  sample_mean = mean(as.array(X))
+  expect_equal(sample_mean, 2, tolerance=1e-2)
+})
+
+test_that("mx.rnorm", {
+  X <- mx.rnorm(shape=50000, mean=5, sd=1, ctx=mx.ctx.default())
+  sample_mean = mean(as.array(X))
+  sample_sd = sd(as.array(X))
+  expect_equal(sample_mean, 5, tolerance=1e-2)
```

Comment:
From 50k samples, the variance of the mean is `1/sqrt(50000)`, roughly 0.0044, this means that the probability of failing the `1e-2` tolerance from a correct implementation is around 2.5%. Considering the number of test being run, this probability is too high and may annoy the others. 

A simple fix will be setting `sd=0.1`.
The `runif` case has the same issue. The fix can be `max=1`, and expect equal to `0.5`.